guide/index.html

```diff
@@ -0,0 +1,230 @@
+
+    <!-- Navigation -->
+    <nav id="mainNav" class="navbar navbar-default navbar-fixed-top navbar-custom">
+        <div class="container">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header page-scroll">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
+                </button>
+                <a class="navbar-brand" href="#page-top">ClimateAction.tech</a>
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right">
+                    <li class="page-scroll">
+                        <a href="#about">About</a>
+                    </p></li>
+                    <li class="hidden">
+                        <a href="#page-top"></a>
+                    </p></li>
+                    <li class="page-scroll">
+                        <a href="#campaigns">Accelerator</a>
+                    </p></li>
+                    <li class="page-scroll">
+                        <a href="#portfolio">Projects</a>
+                    </p></li>
+                    <li class="page-scroll">
+                        <a href="#buzz">Buzz</a>
+                    </p></li>
+                    <li class="page-scroll">
+                        <a href="#contact">Join Us</a>
+                    </p></li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+
+        <!-- /.container-fluid -->
+    </nav>
+
+  
+    <!-- CAT Guide Section -->
+
+	<header>
+        <div class="container" id="maincontent" tabindex="-1">
+            <div class="row">
+                <div class="col-lg-12">
+                    <div class="intro-text">
+                        <h1 class="name">Climate &#43; Tech Guide</h1>
+                        <hr class="star-light">
+                        <span class="skills">Alpha 1.0 &mdash; Climate Action Resources for Tech Professionals</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <section id="about-guide" class="sub-section">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-10 col-lg-offset-1">
+                    <div class="text">
+                        <h2>What this is</h2>
+                        <p>This project is an ongoing effort to provide guidance to tech professionals on navigating the climate crisis. It is designed to be useful to people in and outsode of tech firms who work in tech, and to help you leverage your position and navigate to make positive change within your company, organization or in general society.</p>
```

Comment:
I think this is a typo here - _outsode_ vs _outside_

Reply:
Fixed in 5e55c4a47862bb8e7af5195b7ba639a4dffaf7d3